website/pages/welcome.js

```diff
@@ -46,7 +46,7 @@ class Welcome extends React.Component {
             <li><Anchor href="https://github.com/dracula/dracula-ui">GitHub</Anchor></li>
             <li><Anchor href="https://discord.gg/vyA9AdqdbK">Discord</Anchor></li>
             <li><Anchor href="https://github.com/dracula/dracula-ui/tree/master/design">Figma</Anchor></li>
-            <li><Anchor href="https://github.com/dracula/dracula-ui/blob/master/LICENSE.md">License</Anchor></li>
+            <li><Anchor href="https://github.com/dracula/dracula-ui/blob/master/LICENSE">License</Anchor></li>
```

Comment:
Branch was renamed to `main`
```suggestion
            <li><Anchor href="https://github.com/dracula/dracula-ui/blob/main/LICENSE">License</Anchor></li>
```